docs/developer-guide/views.md

```diff
@@ -55,29 +55,6 @@ Whenever `viewState` updates, the view creates a new viewport under the hood. Ty
 > If you are using the Deck canvas as an [overlay on a base map rendered by another library](../get-started/using-with-map), you may need to update the viewport using the API provided by that library rather than by deck.gl.
 
 
-```js
-import {Deck, MapView} from '@deck.gl/core';
-
-const deck = new Deck({
-  ...
-  views: [new MapView()],
-  onClick: ({layer, object}) => {
-    if (layer) {
-      // The viewport is a WebMercatorViewport instance
-      const {viewport} = layer.context;
-      const {longitude, latitude, zoom} = viewport.fitBounds([
```

Comment:
Learning how to use `fitBounds` is a useful concept. Do we cover it somewhere else?

Reply:
I can move these to the `WebMercatorViewport` doc. The method does not exist on other viewport types.